src/agent/api/testmanagementapi.ts

```diff
@@ -23,18 +23,18 @@ export class TestManagementApi implements ifm.ITestManagementApi {
     }
     
     public createTestRun(testRun: ifm.TestRun, onResult: (err: any, statusCode: number, publishedTestRun: ifm.TestRun) => void): void {
-        this.restClient.create('/_apis/test/runs', "3.0-preview.1", testRun, null, onResult);
```

Comment:
There is another PR that should go in today for the generated test management APIs.  You need to wait on that and not change the hand code one.  Here it is: https://github.com/Microsoft/vso-agent/pull/64  I've asked Scott to prioritize getting that in this morning